hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/table/BaseUIEntityEvent.java

```diff
@@ -41,4 +79,27 @@ public BaseEntityEventType getEventType() {
         return eventType;
     }
 
+    /**
+     * Check if the remove event is the same this ui event. Then maybe you can
+     * skip the remote event because is already executed.
+     * 
+     * @param tenantAwareEvent
+     *            the remote event
+     * @return <true> match <false> not match
```

Comment:
This is not a valid JavaDoc format. Better go for `{@code true}` as described here => https://blogs.oracle.com/darcy/entry/javadoc_tip_code_and_literal